student.rb

```diff
@@ -14,8 +14,8 @@ def play_hooky
     # rubocop:disable RuleByName
   end
 
-  def classrooms(classroom)
-    @classroom = classroom
-    classroom.student.push(self) unless classroom.student.include?(self)
-  end
+#   def classrooms(classroom)
+#     @classroom = classroom
+#     classroom.student.push(self) unless classroom.student.include?(self)
+#   end
```

Comment:
- [ ] it would be better if you remove the unused commented code in order to keep the file clean and pro